### PR TITLE
feat: release docker sbt image 1.3.5

### PR DESCRIPTION
**Pull request by lili2311**
- [ ] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?

Docker image for snyk cli and sbt 1.3.5

### CODE REVIEW

File: docker/Dockerfile.sbt-1.3.5 [hunk 1]
```diff
@@ -0,0 +1,44 @@
+FROM openjdk:8-jdk-slim
```
Comment (gitphill) [inline docker/Dockerfile.sbt-1.3.5]: when should we move to higher java versions?

Comment (lili2311) [reply docker/Dockerfile.sbt-1.3.5]: Can try 11 yep!

Comment (lili2311) [reply docker/Dockerfile.sbt-1.3.5]: added 11 too

**Comment by garethr** [comment]
Just to note that the [experimental images](https://github.com/snyk/snyk-images) have a Scala image `snyk/snyk:scala` available.

**Pull request closed without merging**